src/interfaces/IOptionSettlementEngine.sol

```diff
@@ -254,14 +258,6 @@ interface IOptionSettlementEngine {
      */
     function setFeeTo(address newFeeTo) external;
 
-    /**
-     * @notice Retrieves the optionId for the keccak256 hash of an Option struct
-     * with settlementSeed set to 0 at time of hashing.
-     * @param hash The keccak256 hash of an Option struct.
-     * @return optionId The tokenId if it exists, else 0.
-     */
-    function hashToOptionToken(bytes32 hash) external view returns (uint256 optionId);
```

Comment:
This function will likely still be useful, unless you think it can always be computed off chain.